process/tech-due-diligence-guidelines.md

```diff
@@ -0,0 +1,103 @@
+# Technical Due Diligence Guidelines
```

Comment:
This is broader than just technical due diligence. I'd remove the "technical" from the title.

Reply:
@bgrant0607 @monadic That's a salient point.  Up until now we have been referring to this as a technical due diligence.  Are any other due diligences (e.g. legal, financial etc) done as part of the onboarding evaluation?  Or is this, as you imply, 'the' due diligence?  If so, agreed, the title should be changed, and we might want to flesh out the missing bits.  If not, we probably need references to the other due diligences that need to take place.  I'm pretty nervous about engineers doing legal, financial etc due diligences.  How do we want to handle this?

Reply:
CNCF staff are doing the legal due diligence, which is basically that the project and its dependencies are under approved licenses and that a trademark (if it exists) will be transferred to CNCF. We're happy to have TOC Contributors review our work here.

I don't think there is any business due diligence, since (at least in principle) we're fine accepting a project with maintainers that are working on it in their spare time.

But I would be fine removing Technical from the title here.

Reply:
Thanks @dankohn @bgrant0607.  I've removed 'Technical' and referred to the CNCF staff legal due diligence.